### PR TITLE
DM-7697: meas_mosaic crashes on build b/c missing pex_logging in .table and .cfg

### PR DESCRIPTION
**Pull request by erykoff**
Add pex_logging so meas_mosaic compiles.  Also update path to eigen.h

### CODE REVIEW

File: ups/meas_mosaic.table [hunk 1]
```diff
@@ -1,6 +1,8 @@
 setupRequired(afw)
 setupRequired(numpy)
+setupRequired(ndarray)
```
Comment (timj) [inline ups/meas_mosaic.table]: This table file is missing python, boost, gsl, eigen and cfitsio if you compare the list to the `.cfg` file.

**Comment by timj** [comment]
@laurenam does `meas_mosaic` really use `cfitsio`? I see the include file being used but I worry about direct access to FITS files bypassing the butler.

**Comment by erykoff** [comment]
Good question.  There's an include there, but the only fits access I see is via afw::table::BaseCatalog .  Should I remove all references to fitsio and test?

**Comment by jdswinbank** [comment]
If you can simply drop the fitsio include, that would be great. If it takes any surgery beyond that, please don't bother for now: we know that meas_mosaic isn't a long term solution, so I'd prefer not to do any work on it beyond the bare minimum.

**Comment by timj** [comment]
I'm with @jdswinbank on this. If removing that one include file, and removing cfitsio from the .cfg and .table files, means that everything still builds fine then it would be great to include those in this ticket. If removing those things breaks everything just leave it as is.

**Comment by erykoff** [comment]
Makes sense.  I'm waiting for my previous build to finish, then I'll test that.  If it works, great! If not, I'll let it go.

**Comment by laurenam** [comment]
@erykoff, do you also have a dataset handy on which you could run `meas_mosaic` (e.g. the output from Jim's tutorial)?  That would be a great test beyond it just building.  And you should specify on your command line that you want the diagnostics to be produced (run mosaic.py --help for the details) as that's a place where fits tables are being produced & written to disk (I do think you are right about them all going through afw and that the import is historical, but best to check!)

**Comment by laurenam** [comment]
Sounds good :)

**Comment by erykoff** [comment]
Okay, I have checked (finally, now that obs_subaru works again), that diagnostics work just fine without the unnecessary cfitsio inclusion.

**Comment by laurenam** [comment]
Great.  Good to go!

**Pull request merged**